MAINTAIN.md

```diff
@@ -158,6 +158,8 @@ These dependencies are "vendored" (inlined), we must update the sources manually
     * Refer to [`LuaCATS/lpeg`](https://github.com/LuaCATS/lpeg) for updates.
 * `runtime/lua/vim/re.lua`: LPeg regex module.
     * Vendored from LPeg. Needs to be updated when LPeg is updated.
+* `runtime/lua/vim/_meta/re.lua`: docs for LPeg regex module.
```

Comment:
I thought the files under `_meta` were autogenerated?

Reply:
Yes -- so far. But that's not what makes them live under _meta (it's that they contain no code).